### ISSUE TITLE
System Error when running Ganache 1.1.0 on linux

### ISSUE THREAD

**Issue by coffeeprince** [trufflesuite/ganache#838]
PLATFORM: linux
GANACHE VERSION: 1.1.0

EXCEPTION:
```
Error: listen EADDRINUSE 127.0.0.1:8545
    at Object.exports._errnoException (util.js:1024:11)
    at exports._exceptionWithHostPort (util.js:1047:20)
    at Server.setupListenHandle [as _listen2] (net.js:1319:14)
    at listenInCluster (net.js:1367:12)
    at doListen (net.js:1493:7)
    at _combinedTickCallback (internal/process/next_tick.js:141:11)
    at process._tickCallback (internal/process/next_tick.js:180:9)
```

APPLICATION LOG:
```
T+0ms: Starting server with initial configuration: {"hostname":"127.0.0.1","port":8545,"network_id":5777,"total_accounts":10,"unlocked_accounts":[],"vmErrorsOnRPCResponse":true}
```

**Comment by Shonminh** [comment]
see #665 .

**Comment by mallapurbharat** [comment]
I too faced the same issue when using Ganache 1.2.0.
Closing all instances of Ganache did not release the port. However, restarting the system cleared all open ports, and fixed the problem! I've shifted back to Ganache 1.1.0 as there are many issues with Ganache 1.2.0 and 1.2.1 as well.

**Comment by seesemichaelj** [comment]
@mallapurbharat I'm sorry you're still having issues with this problem in 1.2.x. The port staying open was hopefully solved in 1.2.0 (via #665), but it's possible there are other reasons that are causing the port to be open.

We are not experiencing this issue anymore in 1.2.x. Can you please elaborate what you were doing which caused this issue in 1.2.x? When you closed the instances of Ganache, did you force quit each process? Thanks!

**Comment by mallapurbharat** [comment]
I'm not facing this issue now. I think that I was running a series of tests using 1.1.0, 1.2.0 and 1.2.1 for checking bugs #841, #842. At that time I came across this issue. However, once I restarted my system (or killed all running ganache processes using System Monitor on Ubuntu), it resolved this issue. 

Sorry for the late update. I think I missed out on your query while closing the other reported bugs from my side.
